### PR TITLE
single glossary with tags

### PR DESCRIPTION
**Pull request by womullan**

### CODE REVIEW

File: bin/generateAcronyms.py [hunk 1]
```diff
@@ -75,6 +75,70 @@ def _parse_line(line):
...
+            hasTag = False
+            if tags:
+                tagset.update(tags.split())
+                hasTag = tagset.intersection(utags)
+            if acr is None:
+                continue
+
+            if acr not in definitions:
+                definitions[acr] = set()
+            # Ok lets try to do something with Tags .. like if its tagged take this one
```
Comment (timj) [inline bin/generateAcronyms.py]: nitpick: There are lots of typos in these comments.

File: bin/generateAcronyms.py [hunk 2]
```diff
@@ -75,6 +75,70 @@ def _parse_line(line):
...
+        name, definition, tags etc ..
+    tags : `set'
+        List of tags supplied by user to decide which definintion to keep when there are many.
+    init : `dict`
+        Initial definitions to augment with the content from this file.
+
+    Returns
+    -------
+
+    def acronyms : `dict`
```
Comment (timj) [inline bin/generateAcronyms.py]: nitpick: remove "def"

File: bin/generateAcronyms.py [hunk 3]
```diff
@@ -75,6 +75,70 @@ def _parse_line(line):
...
+    """
+
+    if init is None:
+        definitions = {}
+    else:
+        definitions = init.copy()
+
+    lc = 0
+    with open(filename, "r") as fd:
+        reader = csv.reader(fd, delimiter=',', quotechar='"')
```
Comment (timj) [inline bin/generateAcronyms.py]: This is all fine but I wonder if we could tweak the CSV to make it readable by `astropy.table.Table.read()` with format "csv" so you would get a table with column names read directly from the file.

File: bin/generateAcronyms.py [hunk 4]
```diff
@@ -561,13 +615,19 @@ def update(texfiles):
     parser.add_argument('-u', '--update', action='store_true',
                         help="""Update files to put \\gls on acronyms and
                                 glossary entries .""")
-
+    parser.add_argument('-t', '--tags',
+                        help="""Space seprated list of tags between quotes
```
Comment (timj) [inline bin/generateAcronyms.py]: "seprated"

File: bin/generateAcronyms.py [hunk 5]
```diff
@@ -561,13 +615,19 @@ def update(texfiles):
     parser.add_argument('-u', '--update', action='store_true',
                         help="""Update files to put \\gls on acronyms and
                                 glossary entries .""")
```
Comment (timj) [inline bin/generateAcronyms.py]: nitpick: space before period.

File: bin/generateAcronyms.py (hunk 1 above)
Comment (womullan) [reply bin/generateAcronyms.py]: ok ran aspell through it .. i can not do more

File: bin/generateAcronyms.py (hunk 2 above)
Comment (womullan) [reply bin/generateAcronyms.py]: ok

File: bin/generateAcronyms.py (hunk 3 above)
Comment (womullan) [reply bin/generateAcronyms.py]: perhaps for another day ..

File: bin/generateAcronyms.py (hunk 4 above)
Comment (womullan) [reply bin/generateAcronyms.py]: aspell got it

**Pull request merged**